### PR TITLE
added contributor image in readme

### PR DESCRIPTION
**Pull request by AnshuPathak-88825**

### CODE REVIEW

**Comment by AnshuPathak-88825** [comment]
PR for issue #21

File: README.md [hunk 1]
```diff
@@ -45,7 +45,11 @@ TextUtils is licensed under the MIT License - see the [LICENSE](LICENSE) file fo
 <td align="center"><a href="https://github.com/Susmita-Dey"><img src="https://avatars.githubusercontent.com/u/79099734?v=4" width="100px;" alt=""/><br /><sub><b>Susmita Dey</b></sub></a></td>
   </tr>
 </table>
+<h2>Project Contributors⭐</h2>   <a name = "contributors"></a>
```
Comment (Susmita-Dey) [inline README.md:48]: @AnshuPathak-88825 Remove the `<a name = "contributors"></a>` from this line

File: README.md [hunk 2]
```diff
@@ -45,7 +45,11 @@ TextUtils is licensed under the MIT License - see the [LICENSE](LICENSE) file fo
...
 
+<a href="https://github.com/Susmita-Dey/TextUtils/graphs/contributors" align="center">
+  <img src="https://contrib.rocks/image?repo=Susmita-Dey/TextUtils" /> 
+</a>
 <p align="center">
```
Comment (Susmita-Dey) [inline README.md:53]: Add a <hr> tag above this line

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]

**Comment by AnshuPathak-88825** [comment]
check it again

**Comment by Susmita-Dey** [comment]
@AnshuPathak-88825 Add the `hr` tag

**Comment by AnshuPathak-88825** [comment]
done @Susmita-Dey check it again

**Review by Susmita-Dey — APPROVED** [review]
LGTM

**Pull request merged**